concordium-consensus/tests/globalstate/GlobalStateTests/BlockHash.hs

```diff
@@ -197,13 +197,17 @@ tests = do
                 defaultHash `shouldNotBe` hash'
 
             specify "Hash of emptyPersistentTransactionOutcomes (TOV0) is hash of emptyTransactionOutcomesV0" $
-                runDummyHashMonad (getHashM (emptyPersistentTransactionOutcomes @'TOV0))
-                    `shouldBe` getHash @(TransactionOutcomesHashV 'TOV0) emptyTransactionOutcomesV0
+                runDummyHashMonad @'P4 (getHashM (emptyPersistentTransactionOutcomes @'TOV0))
+                    `shouldBe` emptyTransactionOutcomesHashV0
 
             specify "Hash of emptyPersistentTransactionOutcomes (TOV1) is emptyTransactionOutcomesHashV1" $
-                runDummyHashMonad (getHashM (emptyPersistentTransactionOutcomes @'TOV1))
+                runDummyHashMonad @'P6 (getHashM (emptyPersistentTransactionOutcomes @'TOV1))
                     `shouldBe` emptyTransactionOutcomesHashV1
 
             specify "Hash of emptyPersistentTransactionOutcomes (TOV2) is emptyTransactionOutcomesHashV2" $
-                runDummyHashMonad (getHashM (emptyPersistentTransactionOutcomes @'TOV2))
+                runDummyHashMonad @'P9 (getHashM (emptyPersistentTransactionOutcomes @'TOV2))
                     `shouldBe` emptyTransactionOutcomesHashV2
+
+-- specify "Hash of emptyPersistentTransactionOutcomes (TOV3) is emptyTransactionOutcomesHashV3" $
```

Comment:
We need to merge the p10 branch into the feature branch before we turn this test on.